simphony/io/tests/test_data_container_table.py

```diff
@@ -1,15 +1,15 @@
 import unittest
 from contextlib import closing
 
-import tables
-
 from simphony.io.data_container_description import Record
 from simphony.io.data_container_table import DataContainerTable
 from simphony.io.tests.abc_data_container_table_check import (
     ABCDataContainerTableCheck)
 
+try:
+    import tables
 
-class CustomData(tables.IsDescription):
+    class CustomData(tables.IsDescription):
```

Comment:
I think that the best solution is to setup a `load_tests` function in `__init__.py` of the `tests` directory and use

```python
python -m unittest discover -p tests
```

see http://stackoverflow.com/questions/12487532/how-do-i-skip-a-whole-python-unittest-module-at-run-time